filters/SkewnessBalancingFilter.cpp

```diff
@@ -83,7 +83,8 @@ std::set<PointId> SkewnessBalancingFilter::processGround(PointViewPtr view)
     }
 
     PointId j(0);
-    for (PointId i = view->size() - 1; i >= 0; --i)
+    const auto size(view->size());
```

Comment:
Umm, I'm not a fan.  It makes you think about wrapping.  How about just writing the loop differently?

```
while
{
} while (--i != 0)
```

Reply:
If the view is empty we can just early exit at the top.